abci/example/kvstore/kvstore.go

```diff
@@ -139,7 +139,7 @@ func (app *Application) Info(context.Context, *types.InfoRequest) (*types.InfoRe
 	}
 
 	var defaultLanePriority uint32
-	if len(app.lanes) > 0 {
```

Comment:
Why this change?

Reply:
I pushed for it because we explicitly pass `nil` as lanes in some places but I realize that it won't throw any error in that case, the condition will just be false. We can revert the change.

Reply:
I think it's more "goish" to use `len` unless there's a good reason not to use it

Reply:
yes, my bad, lots of c/c++ ptsd